app/Http/Controllers/AdminsController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use Hash;
+use Validator;
+
+class AdminsController extends Controller
+{
+
+    function NewAdmin (Request $r)
+    {
+    	$validation=Validator::make($r->all(),
+    	[
+    		"name"=>"required",
+    		"surname"=>"required",
+    		"email"=>"required|email|unique:users,email",
+    		"phone"=>"nullable|numeric",
+    		"password"=>"required|min:6",
+    		"config_password"=>"required|same:password",
```

Comment:
why did you name it config_password, what is the purpose? is this for password confirmation?